techmatters/flex-plugins
# CHI-1016: Terraform Provisioners for API Keys & Flex Service Config

Primary Reviewer: @murilovmachado 
## Description

* Added a small command to `twilioResources` that wraps POSTing updated flex configuration for use by the provisioner, mainly to make it platform independent & allow specifying the payload as an env var (easier than escaping JSON for the command line!)
* Wraps the previous manual workarounds to create API keys (and associated SSM parameters) and to update Flex Service Configuration to be terraform provisioners. This means they should run automatically as part of a `terraform apply`
* Updates /twilio-iac/README.md to reflect the simplified process
* Added an environment variable flag to workflowGenerator script to ignore the result of the comparison check (it's always failing on scripts I know are correct)

### Checklist
- [ X ] Corresponding issue has been opened
- [ N/A ] New tests added
- [ N/A ] Strings are localized

### Related Issues
As part of CHI-1016

### Verification steps

* Follow the steps to set up a new helpline in the README.md
* Verify that the 2 Twilio API keys and 3 associated SSM parameters have been set up correctly
* Verify the flex service configuration is correct using the REST API
